src/Sylius/Bundle/CartBundle/spec/Form/Type/CartItemTypeSpec.php

```diff
@@ -37,12 +38,18 @@ function it_is_a_form_type()
         $this->shouldImplement(FormTypeInterface::class);
     }
 
-    function it_builds_form_with_quantity_field(FormBuilder $builder)
+    function it_builds_form_with_quantity_field($orderItemQuantityDataMapper, FormBuilder $builder)
     {
         $builder
             ->add('quantity', 'integer', Argument::any())
+            ->willReturn($builder)
             ->shouldBeCalled()
```

Comment:
Anyway, isn't it more natural to call `willReturn()` / `willThrow()` at the end of method chain?